app/controllers/store_controller.rb

```diff
@@ -6,4 +6,8 @@ class StoreController < ApplicationController
   def index
     @products = Product.order(:title)
   end
+
+  def categories
```

Comment:
this should be in categories_controller

Reply:
Done it in another branch, will merge it after review